pom.xml

```diff
@@ -256,7 +257,7 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-simple</artifactId>
-            <scope>provided</scope>
+            <scope>compile</scope>
```

Comment:
I would rather have this scope runtime, as definitely not needed in the compile class path.

Reply:
Feel free to adjust if it is still working then

Reply:
Done in https://github.com/apache/sling-org-apache-sling-feature-launcher/commit/b55ca9cb6d3019aef854c45e717eb476e874c4d0.